runtime/j9vm/java11vmi.c

```diff
@@ -913,19 +1056,46 @@ JVM_AddModuleExports(JNIEnv * env, jobject fromModule, const char *package, jobj
  * 3) Package is not defined for fromModule's class loader
  * 4) Package is not in module fromModule.
  */
+#if JAVA_SPEC_VERSION >= 15
+void JNICALL
+JVM_AddModuleExportsToAll(JNIEnv * env, jobject fromModule, jstring packageObj)
+#else
 void JNICALL
 JVM_AddModuleExportsToAll(JNIEnv * env, jobject fromModule, const char *package)
+#endif /* JAVA_SPEC_VERSION >= 15 */
 {
 	J9VMThread * const currentThread = (J9VMThread*)env;
 	J9JavaVM const * const vm = currentThread->javaVM;
 	J9InternalVMFunctions const * const vmFuncs = vm->internalVMFunctions;
+#if JAVA_SPEC_VERSION >= 15
+	jboolean isCopy = JNI_FALSE;
+	const char* package = (NULL == packageObj) ? NULL : (*env)->GetStringUTFChars(env, packageObj, &isCopy);
+	if ((NULL != package) && (JNI_FALSE == isCopy)) {
```

Comment:
same here as well

Reply:
Updated to use copyStringToUTF8Helper().